cmd/internal/codegen/parse/crd.go

```diff
@@ -173,6 +174,24 @@ var primitiveTemplate = template.Must(template.New("map-template").Parse(
     {{ if .Format -}}
     Format: "{{ .Format }}",
     {{ end -}}
+    {{ if .EnumValue -}}
+    Enum: {{ .EnumValue }},
+    {{ end -}}
+    {{ if .MaxLength -}}
+    MaxLength: getInt({{ .MaxLength }}),
+    {{ end -}}
+    {{ if .MinLength -}}
+    MinLength: getInt({{ .MinLength }}),
+    {{ end -}}
+    {{ if .MaxItems -}}
+    MaxItems: getInt({{ .MaxItems }}),
```

Comment:
MaxItems, MinItems, UniqueItems don't apply to primitives.  They apply to slices.  We should throw an error if we see them on primitive elements.

Reply:
Likewise, we should probably throw errors if we see things that apply to primitive values on non-primitive fields.